### PR TITLE
Moved to pbr

### PR DESCRIPTION
**Pull request by adir-stratoscale**
@rom-stratoscale please review.

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Adir Gabai <adir@stratoscale.com>
+Eran <eran@stratoscale.com>
```
Comment (rom-stratoscale) [inline AUTHORS]: I think @eran-stratoscale has a sure name :)
Lets try Cohen.

Comment (adir-stratoscale) [inline AUTHORS]: The file was generated automatically. I can change it manually, but i guess it will be overridden. 
@eran-stratoscale did you git-configure your surname?

Comment (eran-stratoscale) [inline AUTHORS]: No, just my first name.

File: Makefile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-all: pep8 pylint tests build
+all: build
```
Comment (eran-stratoscale) [inline Makefile]: all, should be all: pep8 pylint tests & build,
We can move build to the top so it will be the default target.

File: AUTHORS (hunk 1 above)
Comment (adir-stratoscale) [inline AUTHORS]: just like Madonna :)
@eran-stratoscale do you want me to update it manually or just keep it like this?

File: Makefile (hunk 2 above)
Comment (adir-stratoscale) [inline Makefile]: http://stackoverflow.com/questions/2514903/what-does-all-stand-for-in-a-makefile

tl;dr 
all is usually the default target, and it should *build* all products.

Comment (adir-stratoscale) [inline Makefile]: another source: https://www.gnu.org/software/make/manual/html_node/Standard-Targets.html

**Comment by adir-stratoscale** [comment]
i'm starting to think that we don't need to track the AUTHORS and ChangeLog files, since these files are auto-generated. 
@rom-stratoscale @eran-stratoscale what do you think?

**Comment by adir-stratoscale** [comment]
to ship or not to ship? 
i wanna publish new version...

**Comment by rom-stratoscale** [comment]
Ship it. Not sure how how the changelog is auto generated.

**Comment by adir-stratoscale** [comment]
AUTHORS and ChangeLog are auto-generated by pbr.
